packages/graphiql/package.json

```
@@ -31,30 +31,28 @@
     "graphiql.min.css.map"
   ],
   "scripts": {
-    "analyze-bundle": "cross-env NODE_ENV=production CDN=1 ANALYZE=1 yarn webpack -p",
-    "build": "yarn build-clean && yarn build-cjs && yarn build-esm",
-    "build-bundles": "yarn build-bundles-clean && yarn build-bundles-dev && yarn build-bundles-min",
+    "analyze-bundle": "cross-env NODE_ENV=production CDN=1 ANALYZE=1 pnpm webpack -p",
+    "build-bundles": "pnpm build-bundles-clean && pnpm build-bundles-dev && pnpm build-bundles-min",
     "build-bundles-clean": "rimraf 'graphiql.*{js,css}' *.html",
-    "build-bundles-dev": "cross-env NODE_ENV=development CDN=1 yarn webpack --mode development --bail",
-    "build-bundles-min": "cross-env ANALYZE=1 NODE_ENV=production CDN=1 yarn webpack --mode production --bail",
-    "build-cjs": "tsc",
+    "build-bundles-dev": "cross-env NODE_ENV=development CDN=1 pnpm webpack --mode development --bail",
+    "build-bundles-min": "cross-env ANALYZE=1 NODE_ENV=production CDN=1 pnpm webpack --mode production --bail",
     "build-clean": "rimraf esm dist webpack *.html",
     "build-demo": "build-storybook -o ./storybook",
-    "build-esm": "tsc --project ./tsconfig.esm.json",
     "check": "tsc --noEmit",
-    "cypress-open": "yarn e2e-server 'cypress open'",
+    "cypress-open": "pnpm e2e-server 'cypress open'",
     "dev": "cross-env NODE_ENV=development webpack-dev-server --config resources/webpack.config.js",
-    "e2e": "yarn e2e-server 'cypress run'",
+    "e2e": "pnpm e2e-server 'cypress run'",
     "e2e-server": "start-server-and-test 'cross-env PORT=8080 node test/e2e-server' 'http-get://localhost:8080/graphql?query={test { id }}'",
     "storybook": "start-storybook",
     "webpack": "webpack-cli --config resources/webpack.config.js"
   },
   "dependencies": {
     "@graphiql/react": "^0.16.0",
-    "@graphiql/toolkit": "^0.8.1",
+    "@graphiql/toolkit": "^0.8.0",
```

```suggestion
    "@graphiql/toolkit": "workspace:*",
```
All monorepo packages/examples should contain the above

---

hmmm ok, and then they are rewritten on publish? i will have to double check how this works with changesets, because currently this is automatically managed by changesets

---

> and then they are rewritten on publish?

yes